DropBot/Modules/AccessoriesModule.cs

```diff
@@ -23,9 +23,9 @@ public async Task Savage()
         [Summary("Who calls where we drop")]
         public async Task CallIt()
         {
-            var voiceChannels = this.Context.Guild.VoiceChannels;
+            var voiceChannel = this.Context.Guild.GetUser(this.Context.User.Id).VoiceChannel;
```

Comment:
Let's do some error handling in case the user isn't actually in a voice channel. Something like 
```C#
            var user = this.Context.Guild.GetUser(this.Context.User.Id);            
            var channel = user.VoiceChannel;

            if(channel == null)
            {
                await ReplyAsync("Please join a voice channel first.");
                return;
            }
```

Reply:
I suppose that would knock out the other [issue](https://github.com/matta174/DropBot/issues/13)